immutables/_map.pyi

```diff
@@ -0,0 +1,87 @@
+from typing import Any
+from typing import Generic
+from typing import Hashable
+from typing import Iterable
+from typing import Iterator
+from typing import Literal
+from typing import Mapping
+from typing import MutableMapping
+from typing import NoReturn
+from typing import Tuple
+from typing import Type
+from typing import TypeVar
+from typing import Union
+
+
+K = TypeVar('K', bound=Hashable)
+V = TypeVar('V', bound=Any)
```

Comment:
Should the `V` TypeVar set `covariant=True`? That would match https://github.com/python/typeshed/blob/master/stdlib/3/typing.pyi#L413 better.

Reply:
Yes, that seems reasonable.